oidc-authenticator/src/main/java/org/xwiki/contrib/oidc/auth/internal/OIDCClientConfiguration.java

```diff
@@ -288,7 +288,7 @@ public ClaimsRequest getClaimsRequest()
 
         // UserInfo claims
         List<String> userinfoclaims = getUserInfoClaims();
-        if (idtokenclaims != null && !userinfoclaims.isEmpty()) {
+        if (userinfoclaims != null && !userinfoclaims.isEmpty()) {
```

Comment:
Actually that test is not really necessary since getUserInfoClaims should never return null in practice but it sure is better than the previous version :)